### PR TITLE
Added swagger definitions for #14

### PR DESCRIPTION
**Pull request by felixschorer**
- Added swagger definitions for #14 

- Added dummy handler function in `controllers/search.js`

### CODE REVIEW

File: api/swagger/src/paths/search/string.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      required: false
+      type: integer
+      format: int32
+  responses:
+    "200":
+      description: Success
+      schema:
+        $ref: '#/definitions/SearchResults'
+    "404":
+      description: No Results
```
Comment (chaoran-chen) [inline api/swagger/src/paths/search/string.yaml]: 404 for no results? Not 200 with an empty array as result?

Comment (felixschorer) [reply api/swagger/src/paths/search/string.yaml]: That's what was specified in the issue. I was like dafuq as well...

Comment (pfent) [reply api/swagger/src/paths/search/string.yaml]: Ehhhhhhhhhhhhhh, I'd strongly urge @syncall to overthink his specification…

404 usually indicates, that the resource wasn't found, not that the query didn't have any results...

Comment (syncall) [reply api/swagger/src/paths/search/string.yaml]: Oh Ok. Do it like you think it's best. As long as it gives the result and I know how to use it ^^

**Review by kordianbruck — APPROVED** [review]
So this should be fine then?

**Comment by Henni** [comment]
@kordianbruck no, the 404 should be replaced with a 200 response as described by @chaoran-chen and @pfent .

**Comment by kordianbruck** [comment]
@Henni better?

**Review by Henni — APPROVED** [review]

**Pull request merged**